### ISSUE TITLE
Accessibility: ListBox items don't support the ScrollItem pattern

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#932]
ListBox supports the Scroll pattern but ListBox items don't support ScrollItem pattern and don't have ScrollIntoView action.

**Expected result:**
ListBox items should support the ScrollItem pattern.

**Comment by merriemcgaw** [comment]
@vladimir-krestov will this make it in by the end of the month (including testing)? If not, I will move it out to a later Preview release.

**Comment by vladimir-krestov** [comment]
@merriemcgaw 
Initially the issue was fixed in PR #935 and can be checked in. But when working on the issue #1026 it was found that accessibility provider should be switched to UIA to completely address both issues. So at this moment the integrated fix for #932 and #1026 is implemented in #1027. At this moment this is under code review and test. Plan to check-in the fix at the beginning of next week and cancel initial PR #935 as this is not necessary anymore.
